### PR TITLE
HCAP-971 | Disabled form redesign

### PR DESCRIPTION
**Pull request by arranfw**
https://freshworks.atlassian.net/browse/HCAP-971

- There's a bottom navbar in the designs but I didn't put it in because it feels like a separate ticket and it isn't present in other screens
- Didn't add custom styling for the email input to stay consistent with the rest of the app (can be added if we want a second input design in the app)
- Designs were a bit complex with the responsive behaviour so the CSS is a little cumbersome

image
image
image

### CODE REVIEW

File: client/src/components/participant-form/DisabledForm.js [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+    [theme.breakpoints.down('sm')]: {
+      width: '100%',
+    },
+  },
+}));
+
+export const DisabledForm = () => {
+  const classes = useStyles();
+  const handleSubmit = async (values, { setSubmitting }) => {
+    console.log('submitted', { values });
```
Comment (dbayly-freshworks) [inline client/src/components/participant-form/DisabledForm.js:147]: Stay console log here

File: client/src/components/participant-form/DisabledForm.js [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+    await new Promise((resolve) => setTimeout(resolve, 500));
+    setSubmitting(false);
+  };
+
+  return (
+    <Box className={classes.container}>
+      <img src={heroBackground} alt='' className={classes.hero} />
```
Comment (dbayly-freshworks) [inline client/src/components/participant-form/DisabledForm.js:154]: alt is empty here.

**Review by dbayly-freshworks — APPROVED** [review]
Looks ok, two nits.

**Comment by arranfw** [comment]
@dbayly-freshworks 
Empty alt is by design, decorative images should use empty alt tags to explicitly designate them as decorative https://www.w3.org/WAI/tutorials/images/decorative/

Console log is removed in next PR, are you cool with merging?

**Pull request merged**